flang/lib/Lower/Bridge.cpp

```diff
@@ -1174,7 +1174,8 @@ class FirConverter : public Fortran::lower::AbstractConverter {
   }
 
   void genFIR(const Fortran::parser::OmpEndLoopDirective &omp) {
```

Comment:
Do you want to keep this as well? I don't mind to keep it but if it will not be used I would removed it as well.

Reply:
I didn't see the comment on Slack. So just keep it for now and as mentioned by @schweitzpgi mentioned it might be removed later if we see it has no semantic for lowering.